frontend/src/shared/helpers/invalidation-cache/invalidateOrganizationCreated.ts

```diff
@@ -0,0 +1,5 @@
+import { QueryClient } from 'react-query';
+import { GET_ORGANIZATIONS_QUERY } from '@organizations/consts/react-query-keys';
+
+export const invalidateOrganizationCreated = (queryClient: QueryClient): Promise<void> =>
+  queryClient.invalidateQueries(GET_ORGANIZATIONS_QUERY, { refetchInactive: true });
```

Comment:
`{ refetchInactive: true }` co to robi? nie moge znalezc w dokumentacji

Reply:
https://react-query.tanstack.com/reference/QueryClient#queryclientinvalidatequeries
bez tego nie pobiera danych po dodaniu organizacji